### PR TITLE
[Localization] Syrup Bomb translation

### PR DESCRIPTION
**Pull request by Adri1**
## What are the changes the user will see?
None, translation of PR #4441 

## Why am I making these changes?
To have it translated

## What are the changes from a developer perspective?
None

## Checklist
- [X] **I'm using `beta` as my base branch**
- [X] There is no overlap with another PR?
- [X] The PR is self-contained and cannot be split into smaller PRs?
- [X] Have I provided a clear explanation of the changes?

### CODE REVIEW

File: src/locales/pt_BR/battler-tags.json [hunk 1]
```diff
@@ -73,5 +73,7 @@
   "tarShotOnAdd": "{{pokemonNameWithAffix}} tornou-se mais fraco ao fogo!",
   "substituteOnAdd": "{{pokemonNameWithAffix}} colocou um substituto!",
   "substituteOnHit": "O substituto tomou o dano pelo {{pokemonNameWithAffix}}!",
-  "substituteOnRemove": "O substituto de {{pokemonNameWithAffix}} desbotou!"
+  "substituteOnRemove": "O substituto de {{pokemonNameWithAffix}} desbotou!",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
+  "syrupBombLapse": "The sticky syrup slowed down {{pokemonNameWithAffix}}!"
```
Comment (josericardo-fo) [inline src/locales/pt_BR/battler-tags.json:78]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}} foi coberto por um xarope doce e pegajoso!",
  "syrupBombLapse": "O xarope pegajoso desacelerou {{pokemonNameWithAffix}}!"
```

File: src/locales/zh_TW/battler-tags.json [hunk 2]
```diff
@@ -74,5 +74,7 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}}\n截斷尾巴，把它做成了替身！",
   "substituteOnAdd": "{{pokemonNameWithAffix}}的\n替身出現了！",
   "substituteOnHit": "替身代替{{pokemonNameWithAffix}}承受了攻擊！",
-  "substituteOnRemove": "{{pokemonNameWithAffix}}的\n替身消失了……"
+  "substituteOnRemove": "{{pokemonNameWithAffix}}的\n替身消失了……",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
+  "syrupBombLapse": "The sticky syrup slowed down {{pokemonNameWithAffix}}!"
```
Comment (40chyan) [inline src/locales/zh_TW/battler-tags.json:79]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}}陷入了\n滿身糖狀態",
  "syrupBombLapse": "满身糖狀態讓{{pokemonNameWithAffix}}\n變慢了！"
```

File: src/locales/ko/battler-tags.json [hunk 3]
```diff
@@ -74,5 +74,7 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}}[[는]] 꼬리를 잘라 대타로 삼았다!",
   "substituteOnAdd": "{{pokemonNameWithAffix}}의\n대타가 나타났다!",
   "substituteOnHit": "{{pokemonNameWithAffix}}[[를]] 대신하여\n대타가 공격을 받았다!",
-  "substituteOnRemove": "{{pokemonNameWithAffix}}의\n대타는 사라져 버렸다..."
+  "substituteOnRemove": "{{pokemonNameWithAffix}}의\n대타는 사라져 버렸다...",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
+  "syrupBombLapse": "The sticky syrup slowed down {{pokemonNameWithAffix}}!"
```
Comment (EnochG1) [inline src/locales/ko/battler-tags.json:79]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}}\n물엿범벅이 되었다!",
  "syrupBombLapse": "물엿범벅 때문에\n{{pokemonNameWithAffix}}의 스피드가 떨어졌다!"
```

File: src/locales/es/battler-tags.json [hunk 4]
```diff
@@ -74,5 +74,7 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}} se desprende\nde un segmento de su cuerpo y lo usa comoseñuelo!",
   "substituteOnAdd": "¡{{pokemonNameWithAffix}} creó un sustituto!",
   "substituteOnHit": "¡El sustituto recibe daño en lugar del {{pokemonNameWithAffix}}!",
-  "substituteOnRemove": "¡El sustituto del {{pokemonNameWithAffix}} se debilitó!"
+  "substituteOnRemove": "¡El sustituto del {{pokemonNameWithAffix}} se debilitó!",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
```
Comment (Adri1) [inline src/locales/es/battler-tags.json:78]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky candy syrup!",
```

File: src/locales/it/battler-tags.json [hunk 5]
```diff
@@ -74,5 +74,7 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}} si taglia\nla coda e ne fa un sostituto!",
   "substituteOnAdd": "Appare un sostituto di {{pokemonNameWithAffix}}!",
   "substituteOnHit": "Il sostituto viene colpito al posto di {{pokemonNameWithAffix}}!",
-  "substituteOnRemove": "Il sostituto di {{pokemonNameWithAffix}} svanisce!"
+  "substituteOnRemove": "Il sostituto di {{pokemonNameWithAffix}} svanisce!",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
```
Comment (Adri1) [inline src/locales/it/battler-tags.json:78]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky candy syrup!",
```

File: src/locales/ja/battler-tags.json [hunk 6]
```diff
@@ -71,5 +71,7 @@
   "disabledOnAdd": "{{pokemonNameWithAffix}}の\n{{moveName}}\nを　封じこめた！",
   "disabledLapse": "{{pokemonNameWithAffix}}の\nかなしばりが　解けた！",
   "tarShotOnAdd": "{{pokemonNameWithAffix}}は　ほのおに　弱くなった！",
-  "shedTailOnAdd": "{{pokemonNameWithAffix}}は\nしっぽを　切って　みがわりにした！"
+  "shedTailOnAdd": "{{pokemonNameWithAffix}}は\nしっぽを　切って　みがわりにした！",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
```
Comment (Adri1) [inline src/locales/ja/battler-tags.json:75]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky candy syrup!",
```

File: src/locales/de/battler-tags.json [hunk 7]
```diff
@@ -75,5 +75,7 @@
   "substituteOnAdd": "Ein Delegator von {{pokemonNameWithAffix}} ist erschienen!",
   "substituteOnHit": "Der Delegator steckt den Schlag für {{pokemonNameWithAffix}} ein!",
   "substituteOnRemove": "Der Delegator von {{pokemonNameWithAffix}} hört auf zu wirken!",
-  "autotomizeOnAdd": "{{pokemonNameWithAffix}} ist leichter geworden!"
+  "autotomizeOnAdd": "{{pokemonNameWithAffix}} ist leichter geworden!",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky candy syrup!",
+  "syrupBombLapse": "The sticky syrup slowed down {{pokemonNameWithAffix}}!"
```
Comment (CodeTappert) [inline src/locales/de/battler-tags.json:80]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}} wurde in Sirup gehüllt!!",
  "syrupBombLapse": "Der Sirup verlangsamt {{pokemonNameWithAffix}}!"
```

File: src/locales/es/battler-tags.json [hunk 8]
```diff
@@ -74,5 +74,7 @@
   "shedTailOnAdd": "{{pokemonNameWithAffix}} se desprende\nde un segmento de su cuerpo y lo usa comoseñuelo!",
   "substituteOnAdd": "¡{{pokemonNameWithAffix}} creó un sustituto!",
   "substituteOnHit": "¡El sustituto recibe daño en lugar del {{pokemonNameWithAffix}}!",
-  "substituteOnRemove": "¡El sustituto del {{pokemonNameWithAffix}} se debilitó!"
+  "substituteOnRemove": "¡El sustituto del {{pokemonNameWithAffix}} se debilitó!",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky candy syrup!",
+  "syrupBombLapse": "The sticky syrup slowed down {{pokemonNameWithAffix}}!"
```
Comment (DanStevensonx) [inline src/locales/es/battler-tags.json:79]:
```suggestion
  "syrupBombOnAdd": "¡{{pokemonNameWithAffix}} está caramelizado!",
  "syrupBombLapse": "El caramelo pegajoso disminuyó la Velocidad de {{pokemonNameWithAffix}}!"
```

**Review by DanStevensonx — COMMENTED** [review]
Translated to Spanish

File: src/locales/ja/battler-tags.json [hunk 9]
```diff
@@ -72,6 +72,6 @@
   "disabledLapse": "{{pokemonNameWithAffix}}の\nかなしばりが　解けた！",
   "tarShotOnAdd": "{{pokemonNameWithAffix}}は　ほのおに　弱くなった！",
   "shedTailOnAdd": "{{pokemonNameWithAffix}}は\nしっぽを　切って　みがわりにした！",
-  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky, candy syrup!",
+  "syrupBombOnAdd": "{{pokemonNameWithAffix}} got covered in sticky candy syrup!",
```
Comment (Chapybara-jp) [inline src/locales/ja/battler-tags.json:75]:
```suggestion
  "syrupBombOnAdd": "{{pokemonNameWithAffix}}は　あめまみれに　なった！",
```

File: src/locales/ja/battler-tags.json [hunk 10]
```diff
@@ -72,6 +72,6 @@
...
   "syrupBombLapse": "The sticky syrup slowed down {{pokemonNameWithAffix}}!"
```
Comment (Chapybara-jp) [inline src/locales/ja/battler-tags.json:76]:
```suggestion
  "syrupBombLapse": "{{pokemonNameWithAffix}}は　あめまみれで　素早さが　下がった！"
```

**Review by Chapybara-jp — APPROVED** [review]

**Pull request closed without merging**